lib/drives/pages/drive_detail_page.dart

```diff
@@ -181,16 +181,21 @@ class _DriveDetailPageState extends State<DriveDetailPage> {
         widgets.addAll(pendingRidesColumn);
       }
 
-      if (!(_drive!.isFinished || _drive!.cancelled)) {
-        widgets.add(const SizedBox(height: 10));
-        Widget deleteButton = Button(
+      widgets.add(const SizedBox(height: 10));
+      Widget bottomButton;
+      if (_drive!.isFinished || _drive!.cancelled) {
+        bottomButton = Button.error(
+          S.of(context).pageDriveDetailHideDrive,
```

Comment:
If we want to be consistent, the key for this should be `pageDriveDetailButtonHide`